src/SprykerEco/Zed/AvalaraTax/AvalaraTaxConfig.php

```diff
@@ -82,16 +92,46 @@ public function getCompanyCode(): string
         return $this->get(AvalaraTaxConstants::AVALARA_TAX_COMPANY_CODE);
     }
 
+    /**
+     * @api
+     *
+     * @return string
+     */
+    public function getDefaultCustomerCode(): string
+    {
+        return '';
+    }
+
     /**
      * Specification:
      * - Commit the transaction for reporting after the order is placed.
      *
      * @api
      *
-     * @return string
+     * @return bool
+     */
+    public function getIsTransactionCommitAfterOrderPlacementEnabled(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @api
```

Comment:
Add specification:
* Returns the type of document that will be used for calculating taxes before the order is placed at the stage of forming a cart.